net/quagga/src/opnsense/mvc/app/controllers/OPNsense/Quagga/forms/bgp.xml

```diff
@@ -12,6 +12,14 @@
         <hint>Your AS Number here</hint>
     </field>
     <field>
+        <id>bgp.networks</id>
+        <label>Network</label>
+        <style>tokenize</style>
+        <type>select_multiple</type>
+        <allownew>true</allownew>
+        <help>Select the network to advertise, you have to set a Null route via zebra or a static route.</help>
```

Comment:
At the moment, the zebra configuration is empty (only an on / off tick in general). I don't want to change that as this is the job of the system configuration pages (setting static routes).